core-web/libs/sdk/react/src/lib/contexts/PageContext.tsx

```diff
@@ -0,0 +1,37 @@
+import { createContext } from 'react';
+
+import { PageProviderContext } from '../components/PageProvider/PageProvider';
+
+export const PageContext = createContext<PageProviderContext>({
+    containers: {},
+    components: {},
+    layout: {
+        header: false,
+        footer: false,
+        body: {
+            rows: [
+                {
+                    columns: [
+                        {
+                            width: 0,
+                            leftOffset: 0,
+                            containers: []
+                        }
+                    ]
+                }
+            ]
+        }
+    },
+    page: {
+        title: '',
+        identifier: ''
+    },
+    viewAs: {
+        language: {
+            id: ''
+        },
+        persona: {
+            keyTag: ''
+        }
+    }
+});
```

Comment:
this is not needed, make the type `PageProviderContext | null` so you lazy load this context